### PR TITLE
Display Shift Counts for Each Shelter

### LINKED ISSUE #44: Volunteer sign-up: include data about number of volunteers signed-up throughout the day
**Is this task related to a feature or a bug? Please include relevant issue numbers below.**
In the work on issue #34 we created an API endpoint /counts/<SHELTER_ID>?filter_start_after=<TIME>&filter_end_before=<TIME> that returns an array of JSON objects showing how many volunteers are signed up for different time slots throughout the requested time interval. This information should be integrated into the client side application, to inform the user how many volunteers are already available at different times. This will allow the users to decide when their help would be most valuable.

**Describe the solution you'd like**
@AbhilashKotha shared the following prototype with me:
image (2)

That's a great way to visualize the information. Perhaps the chart can cover the entire 24 hours for the selected day (subject to discussion, curious to know what others think on this).

### PR DESCRIPTION
**Pull request by loganwyas**
Fixes #44

**What was changed?**

Each shelter now has a graph associated with them, which displays the shift counts for each time range of the chose date.

**Why was it changed?**

This was changed because the client wants users to be able to view the amount of people who have signed up for shifts.

**How was it changed?**

I added a graph component that takes shift data and prints it out in a colored format.

**Screenshots that show the changes (if applicable):**

image

### CODE REVIEW

**Comment by kate-holdener** [comment]
@loganwyas I suggest rewording "Shift Counts for Time Ranges" to something like "Current Volunteer Counts". Also, I find the colors a bit misleading: red makes me want to sign up for that time slot. @AbhilashKotha do you think it would be better to keep the colors neutral (something like blue for everything)?

**Comment by AbhilashKotha** [comment]
@kate-holdener We can hav same color with different transparency levels. or different shades of blue. light for less occupied and bright for more occupied. Does that sound good?

**Comment by kate-holdener** [comment]
> @kate-holdener We can hav same color with different transparency levels. or different shades of blue. light for less occupied and bright for more occupied. Does that sound good?

That's a cool idea. @loganwyas what do you think?

**Comment by loganwyas** [comment]
@kate-holdener Sounds good to me, I can change it real quick.

**Comment by AbhilashKotha** [comment]
@loganwyas I think the sky-blue shade on the wheat color background is gonna create an accessibility issue. What do you think about this?

image

**Comment by loganwyas** [comment]
@AbhilashKotha Yeah, that would probably be better

**Comment by kate-holdener** [comment]
@loganwyas I merged a PR from Abhilash and now your PR has merge conflicts with main.
The charts are great! Could you adjust the wording a bit. When I hover over a bar in the chart, I see "shifts: 1". Could you change "shifts" to "volunteers" (since that's what we are counting - the number of people working during that time slot). Also, when there are no volunteers, the chart area says "There are no shifts during your selected time range." Can you change this "No volunteers are currently signed up during your selected time range".

**Review by kate-holdener — APPROVED** [review]
This is working well. In mobile view, the chart doesn't fit, I'll file a separate issue on that and we can fix it later.
